### PR TITLE
Define backbone structure for READMEs

### PR DESCRIPTION
**Pull request by balanza**

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -1,5 +1,82 @@
...
+- [Architectural overview](#architectural-overview)
+- [Repositories](#repositories)
+  * [Core repositories](#core-repositories)
+    + [App](#app)
+    + [Infra](#infra)
+    + [Backend](#backend)
+    + [Web](#web)
+    + [Utilities](#utilities)
+    + [Developer tool](#developer-tool)
+  * [Iniziative](#iniziative)
```
Comment (Undermaken) [inline README.en.md:15]:
```suggestion
  * [Initiatives](#iniziative)
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.en.md [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
+    + [Cgn](#cgn)
+    + [Sicilia Vola](#sicilia-vola)
+    + [Green Pass](#green-pass)
+- [Contributing](#contributing)
+- [Licenze](#licenze)
```
Comment (Undermaken) [inline README.en.md:20]:
```suggestion
- [Licenses](#licenze)
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.en.md [hunk 3]
```diff
@@ -1,5 +1,82 @@
...
+
+### Core repositories
+#### App
+*[io-app](https://github.com/pagopa/io-app)* <br/>Mobile application for iOS e Android
+#### Infra
+*[io-infra](https://github.com/pagopa/io-infra)* <br/>Terraform definitions for cloud resources used by `IO`.
+
+*[io-infrastructure-live-new](https://github.com/pagopa/io-infrastructure-live-new)* <br/>
+
+*[gitops](https://github.com/pagopa/gitops)* <br/>CI/CD defintions used by `IO`'s code-review and deploy workflows.
```
Comment (Undermaken) [inline README.en.md:42]:
```suggestion
*[gitops](https://github.com/pagopa/gitops)* <br/>CI/CD definitions used by `IO`'s code-review and deploy workflows.
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,5 +1,82 @@
 # IO
 [Read in english 🇬🇧](README.en.md)
 
+*Indice dei contenuti* <small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>generated with markdown-toc</a></i></small>
```
Comment (Undermaken) [inline README.md:4]:
```suggestion
*Indice dei contenuti* <small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>generato con markdown-toc</a></i></small>
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,5 +1,82 @@
...
+- [Descrizione del progetto](#descrizione-del-progetto)
+- [Overview architetturale](#overview-architetturale)
+- [Repositories](#repositories)
+  * [Core repositories](#core-repositories)
```
Comment (Undermaken) [inline README.md:8]:
```suggestion
  * [Repositories principali](#core-repositories)
```
Not really sure this is a good suggestion

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,5 +1,82 @@
...
+    + [App](#app)
+    + [Infra](#infra)
+    + [Backend](#backend)
+    + [Web](#web)
+    + [Utilities](#utilities)
+    + [Developer tool](#developer-tool)
```
Comment (Undermaken) [inline README.md:14]:
```suggestion
    + [Utilità](#utilities)
    + [Strumenti per lo sviluppo](#developer-tool)
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,5 +1,82 @@
...
+*[io-infra](https://github.com/pagopa/io-infra)* <br/>Le definizioni Terraform delle risorse cloud utilizzate da IO.
+
+*[io-infrastructure-live-new](https://github.com/pagopa/io-infrastructure-live-new)* <br/>
+
+*[gitops](https://github.com/pagopa/gitops)* <br/>Definizione delle CI/CD utilizzate dai repository IO per code-review e deploy
+#### Backend
+TBD
+
+#### Web
+*[io.italia.it](https://github.com/pagopa/io-infra)* Sito istituzionale del progetto, provacy policy, termini e condizioni.
```
Comment (Undermaken) [inline README.md:47]:
```suggestion
*[io.italia.it](https://github.com/pagopa/io-infra)* Sito istituzionale del progetto, privacy policy, termini e condizioni.
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Comment by balanza** [comment]
fix names

**Review by Undermaken — APPROVED** [review]

**Pull request merged**